src/elm/Session/LoggedIn.elm

```diff
@@ -204,6 +218,24 @@ view thisMsg page ({ shared } as model) content =
             viewHelper thisMsg page profile_ model content
 
 
+viewFeedback : Feedback -> Html Msg
+viewFeedback feedback =
+    div
+        [ class "flex justify-center items-center"
+        , style "background-color" (getBackgroundColor feedback.success)
+        ]
+        [ span [ class "ml-auto invisible" ] []
+        , span [ class "flex items-center text-sm h-10 leading-snug text-white font-bold" ]
+            [ text feedback.message ]
+        , span
+            [ class "ml-auto text-lg text-white font-bold mr-5 cursor-pointer"
+            , onClick HideFeedback
+            ]
+            [ text "X"
```

Comment:
Its best we use the `x` icon so we are consistent with the rest of the application.

We can have

```elm
Icons.close ""
```